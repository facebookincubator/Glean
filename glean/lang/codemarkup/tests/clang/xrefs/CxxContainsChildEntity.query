query: codemarkup.cxx.CxxContainsChildEntity _
transform: [gensort, []]
