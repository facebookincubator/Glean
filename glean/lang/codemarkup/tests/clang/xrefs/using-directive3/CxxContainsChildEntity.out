[
  "@generated",
  {
    "key": {
      "parent": {
        "decl": {
          "namespace_": {
            "key": {
              "name": { "key": { "name": { "key": "N0" } } },
              "source": {
                "file": { "key": "test.cpp" },
                "lineBegin": 9,
                "columnBegin": 1,
                "lineEnd": 13,
                "columnEnd": 1
              }
            }
          }
        }
      },
      "child": {
        "decl": {
          "namespace_": {
            "key": {
              "name": {
                "key": {
                  "name": { "key": "N1" },
                  "parent": { "key": { "name": { "key": "N0" } } }
                }
              },
              "source": {
                "file": { "key": "test.cpp" },
                "lineBegin": 10,
                "columnBegin": 1,
                "lineEnd": 12,
                "columnEnd": 1
              }
            }
          }
        }
      }
    }
  },
  {
    "key": {
      "parent": {
        "decl": {
          "namespace_": {
            "key": {
              "name": {
                "key": {
                  "name": { "key": "N1" },
                  "parent": { "key": { "name": { "key": "N0" } } }
                }
              },
              "source": {
                "file": { "key": "test.cpp" },
                "lineBegin": 10,
                "columnBegin": 1,
                "lineEnd": 12,
                "columnEnd": 1
              }
            }
          }
        }
      },
      "child": {
        "decl": {
          "namespace_": {
            "key": {
              "name": {
                "key": {
                  "name": { "key": "N2" },
                  "parent": {
                    "key": {
                      "name": { "key": "N1" },
                      "parent": { "key": { "name": { "key": "N0" } } }
                    }
                  }
                }
              },
              "source": {
                "file": { "key": "test.cpp" },
                "lineBegin": 11,
                "columnBegin": 1,
                "lineEnd": 11,
                "columnEnd": 15
              }
            }
          }
        }
      }
    }
  }
]